# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/sync15/src/client/mod.rs
# Commit: 3326192e8db6
# Full Hash: 3326192e8db60772cb3d6a0f7987dc093c8ef35a
# Author: Sammy Khamis <skhamis@mozilla.com>
# Date: 2023-03-07 16:07:33
# Regressor Bug: 1818349
# File Overlap Count: 23
# Overlapping Files: third_party/rust/sync15/src/client/coll_update.rs, third_party/rust/sync15/src/client/mod.rs, third_party/rust/sync15/.cargo-checksum.json, third_party/rust/sync15/src/client/sync.rs, Cargo.toml
# Description:
#   Bug 1818349 Part 1: Vendor new version of application-services r=markh
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171181
# ==============================================================================

diff -r d7be17cdae33 -r 3326192e8db6 third_party/rust/sync15/src/client/mod.rs
--- a/third_party/rust/sync15/src/client/mod.rs	Tue Mar 07 06:59:50 2023 +0000
+++ b/third_party/rust/sync15/src/client/mod.rs	Tue Mar 07 07:42:14 2023 +0000
@@ -25,7 +25,7 @@
 mod token;
 mod util;
 
-pub(crate) use coll_state::CollState;
+pub(crate) use coll_state::{CollState, LocalCollStateMachine};
 pub(crate) use coll_update::{fetch_incoming, CollectionUpdate};
 pub(crate) use collection_keys::CollectionKeys;
 pub(crate) use request::InfoConfiguration;